docs/sofapython3/examples/component/constraint/CableConstraint/CableSmoothActuation.py

```diff
@@ -13,7 +13,7 @@
 
 def createScene(rootNode):
 
-    rootNode.addObject('RequiredPlugin', pluginName='SoftRobots SofaConstraint SofaDeformable SofaEngine SofaImplicitOdeSolver SofaLoader SofaOpenglVisual SofaPreconditioner SofaSimpleFem SofaSparseSolver SofaTopologyMapping')
+    rootNode.addObject('RequiredPlugin', pluginName='SoftRobots Sofa.Component.SolidMechanics.Spring Sofa.Component.Engine.Select Sofa.Component.ODESolver.Backward Sofa.Component.IO.Mesh Sofa.GL.Component.Rendering2D Sofa.GL.Component.Rendering3D Sofa.GL.Component.Shader Sofa.Component.LinearSolver.Preconditioner Sofa.Component.LinearSolver.Iterative Sofa.Component.Diffusion Sofa.Component.SolidMechanics.FEM.Elastic Sofa.Component.LinearSolver.Direct Sofa.Component.Topology.Mapping Sofa.Component.AnimationLoop Sofa.Component.Constraint.Lagrangian.Correction Sofa.Component.Constraint.Lagrangian.Solver Sofa.Component.Mass Sofa.Component.StateContainer Sofa.Component.Topology.Container.Constant Sofa.Component.Topology.Container.Dynamic Sofa.Component.Visual Sofa.Component.Mapping.Linear')
```

Comment:
I think it's preferable to write a list in Python style. For example: `pluginName=['Sofa.Component.SolidMechanics.Spring', 'Sofa.Component.Engine.Select', ...]`